source/renderer/app/api/utils/apiHelpers.ts

```diff
@@ -21,3 +22,27 @@ export const testSync = (apiMethod: (...args: Array<any>) => any) => {
 // helper code for deferring API call execution
 export const wait = (ms: number): Promise<void> =>
   new Promise((resolve) => setTimeout(resolve, ms));
+export const doesWalletRequireAdaToRemainToSupportTokens = (
+  error: ErrorType,
+  hasAssetsRemainingAfterTransaction?: boolean
+): {
+  requiresAdaToRemainToSupportNativeTokens: boolean;
+  adaToRemain?: number;
+} => {
+  const adaToProceedRegex = new RegExp(
+    /.*I need approximately([\s\d.,]+)ada to proceed.*/
```

Comment:
🚩 Red flag here, I would not rely on stings from CWB. They might change that without knowing. We might need to sync up with them and expect a number?